api/cluster.proto

```diff
@@ -328,7 +328,7 @@ service ClusterManager {
 			summary: "delete clusters"
 		};
 		option (google.api.http) = {
-			delete: "/v1/clusters"
+			delete: "/v1/clusters/delete"
```

Comment:
It's best to unify all the action to `post: "/v1/clusters/{action}"`. So this line could be `post: "/v1/clusters/delete`